src/providers/pdal/qgspdalprovider.cpp

```diff
@@ -334,7 +334,7 @@ QString QgsPdalProviderMetadata::filters( QgsProviderMetadata::FilterType type )
 
     case QgsProviderMetadata::FilterType::FilterPointCloud:
       // TODO get the available/supported filters from PDAL library
-      return QObject::tr( "PDAL Point Clouds" ) + QStringLiteral( " (*.laz *.las)" );
+      return QObject::tr( "PDAL Point Clouds" ) + QStringLiteral( " (*.laz *.las *.LAZ *.LAS)" );
```

Comment:
Would it be possible to make the fix more generic, so that the suffix comparison is not case sensitive? Just in case someone would have `.Las` files :-)

Reply:
I don't think Qt file dialogs support that